user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_parameters/I-DOCSIS_parameters_node_segment.md

```diff
@@ -70,10 +70,30 @@ This page contains an overview of the Node Segment parameters available in the I
 
 - **Percentage CM Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by reflection.
 
-- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by NMTER being out of spec (OOS).
+- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+- **Percentage CM Group Delay or Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+> [!NOTE]
+> Note that **Percentage CM Group Delay OOS** and **Percentage CM Reflection OOS** parameters were afected by NMTTER being out of spec, this was changed since **Percentage CM Group Delay or Reflection OOS** did not have relation between the two previous ones.
+
+The following table provides information about the Group Delay and Reflection status for different cable modems in a given service group. The table consists of three rows: "Group Delay Status," "Reflection Status," and "Group Delay/Reflection Status." Each row represents a specific aspect of the cable modems performance.
```

Comment:
Is this section from line 80 to 95 meant as part of the note? If yes, you should also include these in the note blockquote by adding "> " in front of each line.
The note should also be indented (with two spaces in front of each line) so that it's clear that it belongs under line 75. Or if it doesn't belong there and you'd rather not indent it, it should be moved down to the end of the bulleted list instead. This because the lack of indent now makes it seem as if the list of parameters ends above the note, and then a new list starts below line 95.

Reply:
I didn't add the section to the note because it doesn't match the company look and feel of documentation